docs/web/overview.mdx

```diff
@@ -6,82 +6,66 @@ description: Geocortex Web - Overview
+import ExtensibilitySpectrum from "../../src/components/ExtensibilitySpectrum";
 
-[Geocortex Web](https://www.geocortex.com/products/gxw/) is a next-generation framework for creating sleek and effective GIS applications for a wide variety of browsers. Apps allow users to view and interact with web-based maps and associated data. Geocortex Web can seamlessly display both 2D and 3D map data. Integration with [Geocortex Workflow](../workflow/overview.mdx) ensures that you can build apps for many business processes. Geocortex Web is part of the [Geocortex suite of products](https://www.geocortex.com/products/).
+[Geocortex Web](https://www.geocortex.com/products/gxw/) is a next-generation framework for creating sleek and effective GIS applications for a wide variety of browsers. Apps allow users to view and interact with web-based maps and associated data. Geocortex Web can seamlessly display both 2D and 3D map data. Geocortex Web is part of the [Geocortex suite of products](https://www.geocortex.com/products/).
 
 Geocortex Web apps are created, configured and deployed with [Geocortex Web Designer](https://apps.geocortex.com/webviewer/designer/). Web Designer is an intuitive web application that displays a live preview of Geocortex Web apps as they are being configured.
 
 <DesignerCallout />
 
-## Start with the Concepts
+If the Geocortex Web Designer cannot accomplish what you need, you can manually configure and extend Geocortex Web in a number of ways.
 
-The core element of Geocortex Web is an **Application**. Applications are composed of three main pieces:
+The simplest way to customize your application is through configuration. If you need complex custom behavior and UI, Geocortex Workflow may be able to accomplish your goals through an easy to use interface. Geocortex Workflow can be taken even further through custom extensions to its activity and form element library. Finally, Geocortex Web can be extended through its SDK to create completely custom experiences.
 
-1. [**Layout XML**](configuration-layout.mdx) that represents the visual layout of the application.
+<ExtensibilitySpectrum
+    configLink="#application-configuration"
+    workflowLink="#geocortex-workflow"
+    customWorkflowLink="#custom-workflow-activities-and-form-elements"
+    customCodeLink="#custom-code"
+/>
 
-2. [**App Config JSON**](configuration-app-config.mdx) that configures the application.
+## Application Configuration
 
-3. The Geocortex Web code which renders the layout and powers functionality.
+Every Geocortex Web application has a [**layout**](configuration-layout.mdx), which represents the visual layout of the application, and [**app config**](configuration-app-config.mdx) that configures various behaviors and settings within the application. Layout and app config can be easily edited to customize the appearance and behavior of an an Geocortex Web application without writing custom code.
```

Comment:
Were we trying to draw attention to some of these links by making them bold?

Reply:
Yes that was the idea!